testing/cypress/integration/default-login/default-login.spec.js

```diff
@@ -0,0 +1,97 @@
+    });
+
+    it("should check login form for error if required fields are empty", () =>
+    {
+        cy.getByTestingAttr("submit-login").click();
+        cy.getByTestingAttr("email-login").parent().should("have.class", "error");
+        cy.get(".error-msg").should("contain", "Bitte geben Sie eine gültige E-Mail-Adresse an.");
+        cy.getByTestingAttr("password-login").parent().should("have.class","error");
+        cy.get(".error-msg").should("contain", "Bitte geben Sie Ihr Passwort ein.");
+    });
+
+    it("check for notification after failed login", () =>
+    {
+        cy.getByTestingAttr("email-login").type(`user${new Date().valueOf()}@plentye2etest.de`, { delay: 40 });
+        cy.getByTestingAttr("password-login").type("Test", { delay: 30 });
+        cy.server().route("POST", "/rest/io/customer/login").as("loginUser");
+        cy.getByTestingAttr("submit-login").click();
+        cy.get(".notification-wrapper").children().should("exist").should("have.class", "show");
+        cy.get(".notification-wrapper").children().first().should("contain", "Die Anmeldedaten sind ungültig.");
+    });
+
+    it("should login successfully", () =>
+    {
+        cy.getByTestingAttr("email-login").type("ceres-testing@opentrash.com", { delay: 30 });
+        cy.getByTestingAttr("password-login").type("8#TfhMB@QVd668T", { delay: 30 });
+
+        cy.server().route("POST", "/rest/io/customer/login").as("loginUser");
+
+        cy.getByTestingAttr("submit-login").click();
+
+        cy.wait("@loginUser").then((xhr) =>
+        {
+            expect(xhr.status).to.eql(200);
+
+            cy.wait(1000);
+            cy.getStore().then((store) =>
+            {
+                expect(store.getters.isLoggedIn).to.be.true;
+            });
+        });
+
+        cy.url().should('eq', Cypress.config().baseUrl);
+    });
+
+    it("should check guest login form for error if required fields are empty", () =>
+    {
+        cy.getByTestingAttr("guest-login-button").click();
+        cy.getByTestingAttr("guest-login-input").parent().should("have.class", "error");
+        cy.get(".error-msg").should("contain", "Bitte geben Sie eine gültige E-Mail-Adresse an.");
+    });
+
+    it("should check for valid email at guest login", () =>
+    {
+        cy.getByTestingAttr("guest-login-input").type("ceres-testingopentrash.com", { delay: 30 });
+        cy.getByTestingAttr("guest-login-button").click();
+        cy.getByTestingAttr("guest-login-input").parent().should("have.class", "error");
+        cy.get(".error-msg").should("contain", "Bitte geben Sie eine gültige E-Mail-Adresse an.");
+    });
+
+    it("should check if link to guest login page is working", () =>
+    {
+        cy.getByTestingAttr("guest-login-input").type("ceres-testing@opentrash.com", { delay: 30 });
+        cy.getByTestingAttr("guest-login-button").click();
+
+        cy.server().route("POST", "/");
+        cy.wait(1000);
+        cy.url().should('eq', Cypress.config().baseUrl);
```

Comment:
cy.location("pathname").should("eq", "/");

Reply:
Done